frontend/pages/journey/[id].vue

```diff
@@ -0,0 +1,441 @@
+                  :value="
+                    format(fromDate, 'dd/MM/yyyy') +
+                    ' - ' +
+                    format(toDate, 'dd/MM/yyyy')
+                  "
+                />
+              </div>
+              <div
+                class="w-full xl:col-span-1 2xl:col-span-1 lg:col-span-2 md:col-span-2 absolute"
+              >
+                <SvgStripes class="absolute w-[9.8rem] right-0" />
+              </div>
+            </div>
+          </div>
+          <div
+            class="h-[90%] w-0 rounded-b-r-3xl border-border-gray border-r-2 border-dashed"
+          ></div>
+        </div>
+      </div>
+      <div
+        class="lg:w-1/6 md:w-[30%] h-72 rounded-2xl bg-background border-solid md:visible invisible w-0 max-md:h-0 dark:bg-border-dark"
+      >
+        <div
+          class="h-1/6 bg-border border-x-2 border-t-2 border-border-darker rounded-t-2xl dark:bg-ticket-top-dark-bg dark:border-border-blue-dark"
+        >
+          <div class="w-full flex justify-end items-center h-full">
+            <SvgAirplaneIcon class="w-9 mr-5" />
+          </div>
+        </div>
+        <div class="flex h-5/6">
+          <div
+            class="h-[90%] w-0 rounded-b-l-3xl border-border-gray border-l-2 border-dashed"
+          ></div>
+          <div
+            class="h-full w-full rounded-b-2xl bg-background border-border-gray border-r-2 border-b-2 flex justify-center dark:bg-border-dark dark:border-form-input-dark"
+          >
+            <div class="h-full w-full flex flex-col items-end relative">
+              <img
+                class="absolute right-[50%] top-[25%] z-20 translate-x-[50%] -translate-y-[25%] lg:w-[10rem] md:w-[8rem]"
+                :src="qrcode"
+                alt="QR Code"
+              />
+              <SvgStripes class="absolute w-[9.8rem] right-0" />
+              <button
+                class="absolute right-[50%] top-[80%] translate-x-[50%] -translate-y-[25%] font-bold border-2 border-cta-border h-1/6 w-2/5 rounded-xl hover:bg-cta-bg z-30 bg-background dark:bg-input-dark dark:-translate-y-[5%]"
+              >
+                <T keyName="journey.button.invite" />
+              </button>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="lg:basis-0 md:basis-full basis-0"></div>
+      <div
+        class="lg:w-1/6 md:w-4/5 w-5/6 border-4 rounded-3xl bg-countdown-bg border-solid border-border md:ml-[10%] max-lg:mt-5 dark:bg-surface-dark -ml-3"
+      >
+        <div
+          class="flex flex-wrap lg:flex-col h-full lg:justify-center xs:justify-start justify-center items-center bg-gradient-to-br from-indigo-500 to-indigo-800"
+        >
+          <!-- flip clock container -->
+          <div
+            v-if="hundredsDays <= 0"
+            class="relative font-bold lg:text-6xl text-3xl text-text grid grid-cols-2 gap-x-2 my-2 mx-3 dark:text-white"
+          >
+            <div class="relative bg-black p-2 py-3 rounded-xl">
```

Comment:
bg-black hier und weiter unten wirds nicht geben, ist eine andere Farbe in Figma, bitte ändern damit die Linie überhaupt mal angezeigt wird